tempto-core/src/main/java/com/teradata/tempto/internal/fulfillment/table/hive/HiveTableManager.java

```diff
@@ -60,6 +61,7 @@ public HiveTableManager(
             QueryExecutor queryExecutor,
             FileSystemDataSourceWriter fsDataSourceWriter,
             TableNameGenerator tableNameGenerator,
+            @Named("tests.fs.prefix") String testDataBaseFSPrefix,
```

Comment:
Can you please explain me what it is? It looks we need to update documentation to cover this configuration option.

Reply:
I will nee to update the documentation for this.  The prefix is what is needed for Hive for location storage.  For example, the prefix for S3 would be s3n://[bucketname] ..